_posts/en/2020-07-28-micro-procurment-whats-stopping-us.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title:  "Micro-procurement for digital work: part one"
+ref: working-in-the-open-p1
+lang: en
+author: "Rachel Muston, in collaboration with the IT Strategy team, and procurement experts at ESDC and TBS"
+date: "2020-07-28"
+excerpt: "The Government of Canada (GC) buys a lot of stuff.
+In 2016, the total spent by the GC on goods and services was ~$16 billion.
+From an IT point of view, this spending can include big-ticket items like new data centres and new national applications costing tens of millions of dollars, but can also include low dollar value items like paying someone to build a small prototype over the course of a few days. This is part one of a series looking at how micro-procurements can bring IT and procurement folks together to enable agile, rapid, and scalable digital work."
+---
+<!--markdownlint-disable MD033-->
+This is part one of a two part series looking at micro-procurement for digital work.
+In this first post, we'll look at: some context, the connection between IT transformation and procurement, as well as how, in addition to helping us deploy faster, micro-procurement could increase open source uptake in the GC, improve diversity in GC IT contracting and help GC IT employees acquire new skills by bringing in modern tools and practices.
+
+***
+
+The Government of Canada (GC) buys a lot of stuff.
+In 2016, the total spent by the GC on goods and services was ~$15.9B.
+From an IT point of view, this spending can include big-ticket items like new data centres and new national applications costing tens of millions of dollars, but can also include low dollar value items like paying someone to build a small prototype or a subscription to Survey Monkey.
+
+To ensure that purchasing is working well across the GC, procurement teams are working hard on modernization efforts including the
+development of an [electronic procurement system](https://buyandsell.gc.ca/initiatives-and-programs/the-electronic-procurement-solution/about), enabling [agile procurement](https://www.tpsgc-pwgsc.gc.ca/app-acq/ma-bb/appagile-proagile-eng.html), [social procurement pilots](https://buyandsell.gc.ca/procurement-data/tender-notice/PW-18-00828087) and [contract language simplification](https://www.tpsgc-pwgsc.gc.ca/app-acq/ma-bb/simpcont-simpcont-eng.html).
+Interestingly, Low Dollar Value (LDV) procurements don't seem to make the list for procurement modernization priorities.
+This could be because these procurements are low risk and generally don't make headline news.
+However, for this same reason they provide the perfect opportunity for procurement and IT experimentation.
```

Comment:
_for procurement and IT experimentation_
instead simply?
_for experimentation_